Automattic/wp-calypso
# Tailored Flows: Improve newsletter flow copies

## Proposed Changes

Improve copy on the Newsletter flow.
Copy on the plans step only regards the title, as the content of the page will be addressed in another PR

## Testing Instructions

- Calypso live links
- Check the newsletter flow
- Check the copy

Fixes https://github.com/Automattic/wp-calypso/issues/75016
